### DISCUSSION TITLE
can't find my ipa file after a build

### DISCUSSION THREAD

**Discussion by dabbler** [codemagic-ci-cd/codemagic-docs#2288] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Ionic/Cordova

### Steps to reproduce

My build seems fine.
When looking under Build I see this:
 ▸ Export Succeeded    Successfully exported ipa to build/ios/ipa/App.ipa

But under Publishing:
Skip publishing to App Store Connect: no IPAs or PKGs found

And the ipa file is not shown under Artifacts anywhere for me to download

### Expected results

I expected to be able to download the IPA and/or have the app appear in my testflight tab of my Apple account.

### Actual results

no ipa.  no testflight app appeared

### Build id (optional)

https://codemagic.io/app/6591a24faa68fcd9d47678bc/build/6591d84a0ec45980d63444f7

**Comment by icarusdust** [comment]
Hi @dabbler, the path of IPA under the artifacts section of `codemagic.yaml ` was incorrect, and looks like you have already figured it out. Let me know if otherwise, please.

**Comment by icarusdust** [comment]
Closing this thread, let me know if you have any further questions by replying to this thread or opening a new one.
